config/kubernetes/ug-big-data/teams.yaml

```
@@ -0,0 +1,6 @@
+teams:
+  ug-big-data:
+    description: ""
+    maintainers:
```

Per #566 can you convert to be `members` instead:
```suggestion
    members:
```